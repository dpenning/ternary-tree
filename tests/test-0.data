(4 6 4 18 8 2 14 7 15 5 19 12 15 5 9 0 17 2 2 19)