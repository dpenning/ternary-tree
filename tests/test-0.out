
(((NIL 0 (2) 2 NIL) 2 NIL 4 NIL) 4 (5 5) 6
 ((7) 8 ((9 12) 14 (15) 15 (17)) 18 (19 19))) 
